test/org/sagebionetworks/bridge/services/StormPathUserAdminServiceTest.java

```diff
@@ -131,6 +131,13 @@ public void cannotCreateUserWithSameUsernameOrEmail() {
         } catch(EntityAlreadyExistsException e) { 
             assertEquals("Account already exists.", e.getMessage());
         }
-        
+    }
+
+    @Test
+    public void testDeleteUserWhenSignedOut() {
+        UserSession session = userAdminService.createUser(signUp, study, true, true);
+        authService.signOut(session);
+        // Shouldn't crash
+        userAdminService.deleteUser(study, session.getUser().getEmail());
```

Comment:
Can we validate the user session (or lack thereof) to make sure deleting the user deletes the session?